docs/HowToPlay.md

```diff
@@ -100,3 +100,7 @@ or answer questions for points and powerups
     + blah blah on supply chain, SBOM, OpenC2, OpenChain
 - Google - In-kind Sponsor - Google donated the GCP resources to host this game
     + blah blah on supply chain, SBOM, OpenC2, OpenChain if we can get statements
+
+## Generating of answers
```

Comment:
Good job in the explanation, though in this case, we should consider the end-user, not the developer. 
The end-user is concerned about how they can generate answers , they don't have to know the abstraction behind it . 

You can mention the following:
 - Command used to generate answers
 - How they can use the command
 - What are the options passed to the command
 - What happens if the options are not passed
 - Provide a screenshot of what to expect when they run the command

Reply:
Great feedback @TraceyOnim